### ISSUE TITLE
website has not been deployed since last year

### ISSUE THREAD

**Issue by egonw** [cdk/cdk.github.io#44]
... and I do not immediately see why not.

**Comment by johnmay** [comment]
?

**Comment by egonw** [comment]
> ?

Yeah, also not sure why not. There are changes (commits) in the past year, that are not online. I need to remember how the online website is updated.

**Comment by johnmay** [comment]
Oh right, I will try and find time today to do it

**Comment by johnmay** [comment]
It's not been done in a year as there hasn't been a release and no one has asked me to do it :)

**Comment by johnmay** [comment]
Should be done

**Comment by johnmay** [comment]
OK fixed now, not sure if mastodon verification works but will leave that for you to try

**Comment by egonw** [comment]
> Although now looks like the HTML was broken - whoops should have check that before pushing

oops, sorry about that!

**Comment by egonw** [comment]
> OK fixed now, not sure if mastodon verification works but will leave that for you to try

it does:

![image](https://github.com/user-attachments/assets/4e3204ec-975e-437a-a674-0cd04b108662)

**Comment by egonw** [comment]
@johnmay, sorry, the mastodon icon was not working because of this needed fix: https://github.com/cdk/cdk.github.io/commit/08bb19ce746312ee9be7e30accb3bf60c62e8dc9
